src/Microsoft.AspNet.OData.Shared/Query/Expressions/SelectExpandBinder.cs

```diff
@@ -318,6 +318,19 @@ private Expression ProjectElement(Expression source, SelectExpandClause selectEx
             wrapperProperty = wrapperType.GetProperty("ModelID");
             wrapperTypeMemberAssignments.Add(Expression.Bind(wrapperProperty, Expression.Constant(_modelID)));
 
+            /*
+            // ATTEMPT 1: Attempt to bind the a reference to the model in a similar way to the other properties that exist
+            // in SelectExpandWrapper. This passes the unit test suite but fails 26 cases in E2E suite. This throws
+            // in ODataQueryOptions.cs : 686 (TruncatedCollection<T> truncatedCollection = new TruncatedCollection<T>(queryable, limit);)
+            // and in EnableQueryAttribute.cs : 673 (var result = enumerator.MoveNext() ? enumerator.Current : null;)
+            // Exception: 
+            // NotSupportedException: Unable to create a constant value of type 'Microsoft.OData.Edm.EdmModel'. Only primitive types or enumeration types are supported in this context.
+            //
+            // Sample E2E test that fails: NestedTopSkipOrderByInDollarExpandWorksWithEF
```

Comment:
DerivedAutoExpandNavigationPropertyTest fails and goes through the EnableQueryAttribute.cs